src/configure_config.cmake

```diff
@@ -84,9 +84,11 @@ if (TARGET ch_contrib::datasketches)
 endif()
 if (TARGET ch_contrib::aws_s3)
     set(USE_AWS_S3 1)
+    set(USE_AWS_MEMORY_MANAGEMENT 1)
 endif()
 if (TARGET ch_contrib::google_cloud_cpp)
     set(USE_GOOGLE_CLOUD 1)
+    set(USE_AWS_MEMORY_MANAGEMENT 1)
```

Comment:
I don't think this one is needed here, right?

Reply:
From what I'd gathered, we're using the same S3 client for both S3 and GCS, right? Then this is needed to make this client allocate memory via the custom allocator.